lib/Cake/View/Helper/FormHelper.php

```diff
@@ -1638,7 +1645,12 @@ public function radio($fieldName, $options = array(), $attributes = array()) {
 			$between = '';
 		}
 		if ($legend) {
-			$out = $this->Html->useTag('fieldset', '', $this->Html->useTag('legend', $legend) . $between . $out);
+			if (is_string($fieldset)) {
+				$fieldsetClass = sprintf(' class="%s"', $fieldset);
```

Comment:
I tried implementing it just the way it is done in the method _inputs_, see line [937](https://github.com/xhs345/cakephp/blob/615be3ad1425531160ad7f133b39edfab17151c5/lib/Cake/View/Helper/FormHelper.php#L937) and line [975](https://github.com/xhs345/cakephp/blob/615be3ad1425531160ad7f133b39edfab17151c5/lib/Cake/View/Helper/FormHelper.php#L975). Is it wrong there as well or am I missing something?